front/app/containers/ProjectsShowPage/shared/header/utils.ts

```diff
@@ -0,0 +1,55 @@
+import { UseQueryResult } from '@tanstack/react-query';
+import { TPhases } from 'api/phases/types';
+import { IPCPermissions } from 'api/project_permissions/types';
+import { IProjectData } from 'api/projects/types';
+import { isNilOrError } from 'utils/helperUtils';
+
+export const hasEmbeddedSurvey = (
+  project: IProjectData,
+  phases: TPhases | null
+) => {
+  let hasSurveyPhase = false;
+  if (!isNilOrError(phases)) {
+    phases.map((phase) => {
+      if (phase.attributes.participation_method === 'survey') {
+        hasSurveyPhase = true;
+      }
+    });
+  }
+  if (project.attributes.participation_method === 'survey') {
+    hasSurveyPhase = true;
+  }
+  return hasSurveyPhase;
+};
+
+export const hasSurveyWithAnyonePermissions = (
+  projectPermissions: IPCPermissions | undefined,
+  phasesPermissions:
+    | UseQueryResult<IPCPermissions, unknown>[]
+    | undefined
+    | null
+) => {
+  let hasSurveyWithAnyonePermissions = false;
+
+  if (!isNilOrError(phasesPermissions) && phasesPermissions.length > 0) {
```

Comment:
Can understand why we're doing this, but feels like ideally the backend should be returning this value. One for the future maybe.